ote_sdk/ote_sdk/algo_backends/test_helpers/e2e_test_system.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
can we use the same license header as we use in ote sdk?

Reply:
This is new approved Intel header.
I think we will replace it everywhere, at least it will be in new files.